webapp/src/assetEditor.tsx

```diff
@@ -164,8 +164,15 @@ export class AssetEditor extends React.Component<{}, AssetEditorState> {
         })
     }
 
+    handleKeydown = (e: KeyboardEvent) => {
+        if (e.ctrlKey && (e.key === "s" || e.key === "S")) {
+            this.callbackOnDoneClick();
```

Comment:
In the web app, will pressing ctrl+s close the asset editor?

Reply:
nope! assetEditor.tsx is only used by the ---asseteditor page

Reply:
In the vscode context, would it be possible to:
- honor the user's custom key mapping for the "save" command?
- save when vscode triggers an auto save?
- save when user issues "save all" command.
- use vscode's visual indicator that the doc has unsaved edits (the dot in the tab)?

Reply:
apparently we are moving to auto-save... but this is all theoretically possible. i don't think vscode has hooks for when save is triggered on a custom editor (not that i'm aware of anyways) but we could read it from the shortcuts file and pass it into the webview...

Reply:
actually, we aren't even a custom editor so we definitely don't get save hooks.